main.py

```diff
@@ -1,10 +1,50 @@
 import init_django_orm  # noqa: F401
-
+import json
```

Comment:
optimize your imports

Reply:
I don't understand what I have to do.

Reply:
@serviza Change the order of your imports according to `pep8`. Here `json` should be first.

Reply:
Thanks for explanation)